static/app/utils/profiling/profile/utils.tsx

```diff
@@ -206,11 +206,9 @@ function indexNodeToParents(
 
   // Begin in each root node
   for (let i = 0; i < roots.length; i++) {
-    leafs.push(roots[i]);
-
-    // If the root has no children, continue
+    // If the root is a leaf node, push it to the leafs array
     if (!roots[i].children?.length) {
```

Comment:
This also fixes an edge case where a selected node is the leaf not because we were skipping this, nothing was displaying in the table